src/libraries/Microsoft.PowerFx.Core/Binding/INameResolver2.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.Binding.BindInfo;
+
+namespace Microsoft.PowerFx.Core.Binding
+{
+    internal interface INameResolver2 : INameResolver
```

Comment:
>INameResolver2 [](http://example.com/codeflow?start=23&length=14)

This is a good way to provide a new method on INameResolver in a non-breaking way.